extension/webviews/components/thread/ThreadsViewer.svelte

```diff
@@ -82,5 +82,7 @@
       <Thread {thread} reloadThreads={loadThreads} />
     {/each}
     <hr />
+  {:else}
+    No threads found.
```

Comment:
should we probably inform the user explicitly that no threads are created. sth like `No threads are created yet. Create your first thread.`

Reply:
We should do that. Can be addressed later.